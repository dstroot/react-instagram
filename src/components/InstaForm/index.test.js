import React from 'react';
import { render, cleanup } from '@testing-library/react';

// component to test
import { InstaForm } from '.';

describe('InstaForm', () => {
  it('renders', () => {
    // arange

    // act
    const { asFragment } = render(
      <InstaForm input="hello" handler={jest.fn()} />
    );

    // assert
    expect(asFragment()).not.toBeNull();
    expect(asFragment()).toMatchSnapshot();
  });
});
