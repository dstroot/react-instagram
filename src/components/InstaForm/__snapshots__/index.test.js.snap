// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstaForm renders 1`] = `
<DocumentFragment>
  <form>
    <input
      class="sc-bdVaJa isLXXY"
      name="instagram"
      required=""
      type="text"
      value=""
    />
    <button
      class="sc-bwzfXH ljZSuA"
      type="submit"
      value="Submit"
    >
      Use my Instagram!
    </button>
  </form>
</DocumentFragment>
`;
